packages/fx-core/src/plugins/solution/fx-solution/v2/executeUserTask.ts

```diff
@@ -242,6 +247,9 @@ export function canAddResource(
       sendErrorTelemetryThenReturnError(SolutionTelemetryEvent.AddResource, e, telemetryReporter)
     );
   }
+  if (!projectSetting.solutionSettings) {
```

Comment:
Would be better to use `isExistingTabApp(projectSettings)` to determine whether it's existing web app case.

Reply:
Thank you, fixed.